features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/NewOverview/NewCustomizedStepper.jsx

```diff
@@ -0,0 +1,427 @@
+        setUpdating(true);
+        const promisedUpdate = api.updateLcState(apiId, state);
+        promisedUpdate
+            .then(() => {
+                updateAPI()
+                    .then()
+                    .catch((error) => {
+                        if (error.response) {
+                            Alert.error(error.response.body.description);
+                        } else {
+                            Alert.error('Something went wrong while updating the API');
+                        }
+                        console.error(error);
+                    });
+                Alert.info('Lifecycle state updated successfully');
+            })
+            .finally(() => setUpdating(false))
+            .catch((errorResponse) => {
+                console.log(errorResponse);
+                Alert.error(JSON.stringify(errorResponse.message));
+            });
+    }
+
+    return (
+        <div className={classes.root}>
+            <Stepper alternativeLabel activeStep={activeStep} connector={<ColorlibConnector />}>
+                {steps.map((label) => (
+                    <Step key={label}>
+                        <StepLabel StepIconComponent={ColorlibStepIcon}>
+                            {label === 'Develop' && (
+                                <div>
+                                    <Grid
+                                        container
+                                        direction='row'
+                                        justify='center'
+                                    >
+                                        <Grid item>
+                                            {api ? (
+                                                <CheckIcon className={classes.iconTrue} />
+                                            ) : (
+                                                <CloseIcon className={classes.iconFalse} />
+                                            )}
+                                        </Grid>
+                                        <Grid item style={{ marginBottom: '3px', marginLeft: '3px' }}>
+                                            <Typography variant='h7'>
+                                                <FormattedMessage
+                                                    id='Apis.Details.Overview.CustomizedStepper.Develop'
+                                                    defaultMessage=' Develop'
+                                                />
+                                            </Typography>
+                                        </Grid>
+                                    </Grid>
+                                    {api.type !== 'WEBSUB' && (
+                                        <Grid
+                                            container
+                                            direction='row'
+                                            alignItems='left'
+                                            style={{ marginLeft: '15px' }}
```

Comment:
shall we wrap with a box and use `ml={2}` prop to do the alignments ?

Reply:
Fixed